ios/Codeforces Watcher/Application/features/news/cells/VideoTableViewCell.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  VideoTableViewCell.swift
+//  Codeforces Watcher
+//
+//  Created by Den Matiash on 09.11.2020.
+//  Copyright © 2020 xorum.io. All rights reserved.
+//
+
+import Foundation
+import common
+import UIKit
+
+class VideoTableViewCell: UITableViewCell {
+    
+    private let cardView = CardView()
+
+    private let blogEntryTitleLabel = HeadingLabel()
+    private let userImage = CircleImageView().apply {
+        $0.image = noImage
+    }
+    private let userHandleLabel = SubheadingLabel()
+    private let someTimeAgoLabel = SubheadingLabel()
+
+    private let thumbnailImage = UIImageView(image: UIImage(named: "video_placeholder")).apply {
+        $0.contentMode = .scaleAspectFill
+        $0.clipsToBounds = true
+    }
+    
+    private let horizontalLine = UIView().apply {
+        $0.backgroundColor = Palette.dividerGray
+    }
+    private let explanationLabel = SubheadingLabel().apply {
+        $0.text = "watch_video".localized
+    }
+    private let arrowView = UIImageView(image: UIImage(named: "ic_arrow"))
+    
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setupView()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setupView()
+    }
+
+    private func setupView() {
+        selectionStyle = .none
+        backgroundColor = Palette.white
+
+        buildViewTree()
+        setInteractions()
+    }
+    
+    private func buildViewTree() {
+        contentView.addSubview(cardView)
+
+        [blogEntryTitleLabel, userImage, userHandleLabel, someTimeAgoLabel, thumbnailImage, horizontalLine, explanationLabel, arrowView].forEach(cardView.addSubview)
+    }
+    
+    private func setInteractions() {
+        cardView.edgesToSuperview(insets: UIEdgeInsets(top: 4, left: 8, bottom: 4, right: 8))
```

Comment:
I would be write `cardView.edgesToSuperview(insets: .init(top: 4, left: 8, bottom: 4, right: 8))`.